pkg/scheduler/apis/config/v1alpha2/register.go

```diff
@@ -1,4 +1,6 @@
-/*
+/*if err := AddToScheme(scheme); err != nil {
```

Comment:
lol, what's this?

Reply:
Rogue IDE pasting from flipping tabs too quick. You saw nothing!